docs/symfony-messenger.md

```diff
@@ -0,0 +1,62 @@
+# Using async Symfony Messenger
+
+Add new service to the `compose.yaml`:
+```yaml
+  php-worker:
+  image: ${IMAGES_PREFIX:-}app-php-worker
+  restart: unless-stopped
+  environment:
+    - RUN_MIGRATIONS=false
+  healthcheck:
+    disable: true
+  depends_on:
+    - php
+```
+
+Add new services to the `compose.override.yaml`:
+```yaml
+  php-worker:
+    profiles:
+      - donotstart
+
+  php-worker-async:
```

Comment:
I don't understand how to run `php-worker-async` service in a prod environment.
Please provide an example of how to add the `php-worker-async` service to the `compose.prod.yaml` file.
Thanks.